24w37a ender pearls

EliTheCoder opened the pull request:
Some new methods that have to do with ender pearl chunk loading.

This is my first contribution and I have read CONTRIBUTING.md and CONVENTIONS.md but please let me know if anything looks off.

EliTheCoder committed:
24w37a ender pearls

supersaiyansubtlety commented on mappings/net/minecraft/entity/projectile/thrown/EnderPearlEntity.mapping:
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_xmtgsyuv net/minecraft/entity/projectile/thrown/EnderPearlEntity
+	FIELD f_hbarhomd chunkLoadTimer J
```
```suggestion
	FIELD f_hbarhomd chunkLoadTicks J
```

supersaiyansubtlety commented on mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:
```diff
@@ -183,6 +184,18 @@ CLASS net/minecraft/unmapped/C_mxrobsgg net/minecraft/server/network/ServerPlaye
 		ARG 0 nbt
 		ARG 1 key
+	METHOD m_cqxrhbhc addEnderPearlTicket (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_ynrszrtu;)J
+		COMMENT Adds a chunk ender pearl chunk ticket to the ticket manager.
```
```suggestion
		COMMENT Adds an ender pearl chunk ticket to the ticket manager.
```

supersaiyansubtlety reviewed (changes requested):
looks good, just nitpicks

EliTheCoder committed:
Timer -> Ticks

Co-authored-by: Will <supersaiyansubtlety@gmail.com>

EliTheCoder committed:
Fix typo

Co-authored-by: Will <supersaiyansubtlety@gmail.com>

EliTheCoder commented:
Perfect, thanks

EliTheCoder committed:
Add ender pearl nbt methods

EliTheCoder committed:
Change enderPearls to thrownEnderPearls

It is more descriptive to why ServerPlayerEntity would have a set of
ender pearls.

EliTheCoder committed:
Merge branch '24w37a' of https://github.com/EliTheCoder/quilt-mappings into 24w37a

EliTheCoder committed:
Add and remove from owner methods

EliTheCoder committed:
Projectile set owner from uuid method
